OpsCenter_6.8_Release_Notes.md

```diff
@@ -1,5 +1,25 @@
 # Release notes for OpsCenter
 
+# Release Notes for OpsCenter 6.8.33
+
+## Backup Service
+* Encryption key collision during restore is now prevented and logged. Keys from system tables will no longer be included when restoring. (OPSC-17115)
+
+## Core
+* Added gossip configuration files to diagnostics tarball. (OPSC-17480)
+* Added {{alert_on_no_host_available}} to the {{[cassandra]}} section of the cluster conf. When set to true an alert will be triggered if opscenterd is unable to make a cql connection to the monitored cluster. (OPSC-17398)
+
+## CVE
+* Added {{redirect_host}} config value to the {{[webserver]}} section of {{opscenterd.conf}} to mitigate host header injection vulnerability. (OPSC-17265)
+
+## Monitoring
+* Fixed issue in diagnostic tarball generation when {{diagnostic_tarball_skip_instance_check}} parameter is True. (OPSC-17471)
```

Comment:
From a customer point of view, what was the issue or what did they observe before?

Reply:
The customer was unable to generate a diagnostic tarball. They had issues with instance_check, so we introduced this new parameter  {{diagnostic_tarball_skip_instance_check}} in the previous release to skip instance check before generating a diagnostic tarball. But it had issues reading this parameter values from config file. We fixed it now for this release.